semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala

```diff
@@ -132,6 +135,7 @@ class Main(settings: Settings, reporter: Reporter) {
                 infos.save(out)
               }
             } catch {
+              case e: ClasspathLookup.Error => throw e
```

Comment:
Why give special treatment to this exception?

Reply:
Can we force `lookup` in `Main.process` then, as opposed to doing that in `convertClasspathEntry`? This way this catch clause will become unnecessary.

Reply:
I made `Settings` more strict to avoid this catch